ballot/models.py

```diff
@@ -251,7 +257,9 @@ def refresh_cached_ballot_item_office_info(self, ballot_item, contest_office=Non
 
         return ballot_item
 
-    def retrieve_ballot_item(self, ballot_item_id=0):
+    @staticmethod
+    def retrieve_ballot_item(
```

Comment:
Hi @pbnjcub this works, but we only need to break variables onto a second line when there are enough incoming variables to cause a line wrap.